### DISCUSSION TITLE
Can anyone use UC for this site

### DISCUSSION THREAD

**Discussion by boy0y0** [ultrafunkamsterdam/undetected-chromedriver#1872] [Q&A]
This is a Mandarin site contains pop-up messages. However, the block is quite noticeable and easily seen after closing the pop-up notifications.

https://scr.cyc.org.tw/tp02.aspx?Module=login_page&files=login

i try adding the arguments below - all fails...
    options.add_argument("user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.110 Safari/537.36")
    options.add_argument("--incognito")
    options.add_argument("--disable-blink-features=AutomationControlled")

Copy the message from Cloudflare below. I have no idea how to move forward for debugging, any input would be appreciated.
Troubleshooting
Error Code: 600010
RayId: 87ff9e8db85a48e8
For assistance, please visit Cloudflare's Help Center.
Report a problem
Submitting a report helps us fix bugs in Turnstile.

The widget is too slow

The widget keeps looping

The widget fails sometimes

The widget fails all the time

Other (please specify)
Describe the problem you are encountering...

I consent sending additional troubleshooting data.

**Comment by ganhj99** [comment]
Hi @boy0y0 , have you found any solution for this issue?

**Reply by boy0y0** [reply]
> Hi @boy0y0 , have you found any solution for this issue?

No I have tried all the arguments I can add.
I am not sure where to start studying or debugging as a beginner in both Python and network transit tracing.
